ember-app/app/services/filter_groups/board.js

```diff
@@ -1,7 +1,7 @@
 import Ember from 'ember';
 
 function attr(modelProp, map) {
-  return Ember.computed("model." + modelProp, "model." + modelProp + ".[]", {
+  return Ember.computed("model." + modelProp, {
```

Comment:
Line 35 sets the modelProp to `repos.[]` to begin with, it should stay the same.